Public port note.

Public port note.
Public port note.
Public port note.
Public port note.

Public port note.
Public port note.

Public port note.
Public port note.
Public port note.

Public port note.
Public port note.

--- a/drivers/net/Kconfig
+++ b/drivers/net/Kconfig
@@ -485,4 +485,6 @@
 
 source "drivers/net/hyperv/Kconfig"
 
+source "drivers/net/rtl819x/Kconfig"
+
 endif # NETDEVICES
--- a/drivers/net/Makefile
+++ b/drivers/net/Makefile
@@ -75,3 +75,4 @@
 obj-$(CONFIG_NTB_NETDEV) += ntb_netdev.o
 
 obj-$(CONFIG_FUJITSU_ES) += fjes/
+obj-$(CONFIG_RTL_819X_SWCORE) += rtl819x/
--- a/drivers/char/Makefile
+++ b/drivers/char/Makefile
@@ -60,3 +60,4 @@
 obj-$(CONFIG_TILE_SROM)		+= tile-srom.o
 obj-$(CONFIG_XILLYBUS)		+= xillybus/
 obj-$(CONFIG_POWERNV_OP_PANEL)	+= powernv-op-panel.o
+obj-$(CONFIG_RTL_8211F_SUPPORT)	+= rtl_nfbi/
